src/rabbit_lager.erl

```diff
@@ -407,7 +407,7 @@ generate_handler(Backend, HandlerConfig) ->
                            lists:ukeysort(1, default_handler_config(Backend)))}].
 
 configure_handler_backend(syslog_lager_backend) ->
-    app_utils:load_applications([syslog]);
+    app_utils:start_applications([syslog]);
```

Comment:
Can you use `application:ensure_all_started/1` instead?

`app_utils` does many things behind the scene. That's probably why `syslog` was magically started before #2180 where I removed a lot of its use during RabbitMQ startup (it is still used when stopping RabbitMQ).

Reply:
Done

Reply:
Thanks for noting that @dumbbell